seismicpro/gather/gather.py

```diff
@@ -1245,8 +1246,8 @@ def undo_sdc(self, velocity=None, v_pow=2, t_pow=1):
     #                         Visualization methods                          #
     #------------------------------------------------------------------------#
 
-    @plotter(figsize=(10, 7))
-    def plot(self, mode="seismogram", *, title=None, x_ticker=None, y_ticker=None, ax=None, **kwargs):
+    @plotter(figsize=(10, 7), args_to_unpack="mask")
+    def plot(self, mode="seismogram", mask=None, *, title=None, x_ticker=None, y_ticker=None, ax=None, **kwargs):
```

Comment:
Probably `maskS` to match `event_headerS`. And I'd make it keyword-only.
```suggestion
    def plot(self, mode="seismogram", *, masks=None, title=None, x_ticker=None, y_ticker=None, ax=None, **kwargs):
```

Reply:
done